gradle/libs.versions.toml

```diff
@@ -1,20 +1,18 @@
 [versions]
-kotlin = "1.9.23"
-kotlin-coroutines = "1.8.0"
-ktor = "2.3.10"
+kotlin = "2.0.0"
+kotlin-coroutines = "1.8.1"
+ktor = "2.3.11"
 arrow = "1.2.4"
-kotest = "5.8.1"
+kotest = "5.9.1"
 # https://developer.android.com/jetpack/androidx/releases/compose
-compose = "1.6.6"
-# https://developer.android.com/jetpack/androidx/releases/compose-kotlin
-compose-compiler = "1.5.11" # It's used! Use compatible with Kotlin one
+compose = "1.6.8"
 compose-material3 = "1.2.1"
-glance = "1.0.0"
+glance = "1.1.0"
 hilt = "2.51.1"
 room = "2.6.1"
 androidx-work = "2.9.0"
 kotlinx-collections = "0.3.7"
-paparazzi = "1.3.3"
+paparazzi = "1.3.4"
```

Comment:
Same here, it got upgraded the paparazzi

Reply:
So here 1.3.3? I'll update them

Reply:
Yes

Reply:
Updated ✅ Thanks!